[2025-05-18T20:33:03.109+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: telegram_summary_channel_1137503644.parse_channel_posts scheduled__2025-05-18T20:00:00+00:00 [queued]>
[2025-05-18T20:33:03.114+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: telegram_summary_channel_1137503644.parse_channel_posts scheduled__2025-05-18T20:00:00+00:00 [queued]>
[2025-05-18T20:33:03.115+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-05-18T20:33:03.127+0000] {taskinstance.py:2191} INFO - Executing <Task(DockerOperator): parse_channel_posts> on 2025-05-18 20:00:00+00:00
[2025-05-18T20:33:03.133+0000] {standard_task_runner.py:60} INFO - Started process 285 to run task
[2025-05-18T20:33:03.137+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'telegram_summary_channel_1137503644', 'parse_channel_posts', 'scheduled__2025-05-18T20:00:00+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/dag_generator.py', '--cfg-path', '/tmp/tmpadri2eni']
[2025-05-18T20:33:03.140+0000] {standard_task_runner.py:88} INFO - Job 61: Subtask parse_channel_posts
[2025-05-18T20:33:03.179+0000] {task_command.py:423} INFO - Running <TaskInstance: telegram_summary_channel_1137503644.parse_channel_posts scheduled__2025-05-18T20:00:00+00:00 [running]> on host e5cd1b16e7bb
[2025-05-18T20:33:03.232+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='telegram_summary_channel_1137503644' AIRFLOW_CTX_TASK_ID='parse_channel_posts' AIRFLOW_CTX_EXECUTION_DATE='2025-05-18T20:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-18T20:00:00+00:00'
[2025-05-18T20:33:03.285+0000] {docker.py:359} INFO - Starting docker container from image telegram_parser_task:latest
[2025-05-18T20:33:03.595+0000] {docker.py:429} INFO - [2025-05-18 20:33:03] [INFO] [Parser] - Запуск основного асинхронного процесса парсера (локальный или прямой вызов)...
[2025-05-18 20:33:03] [DEBUG] [Parser] - Using selector: EpollSelector
[2025-05-18T20:33:03.596+0000] {docker.py:429} INFO - [2025-05-18 20:33:03] [INFO] [Parser] - --- Запуск скрипта парсера (Airflow Task) ---
[2025-05-18 20:33:03] [INFO] [Parser] - Целевой канал (Telethon ID): https://t.me/trader_chernyh
[2025-05-18 20:33:03] [INFO] [Parser] - Числовой ID канала (из конфига DAG для XCom/DB): 1137503644
[2025-05-18 20:33:03] [INFO] [Parser] - Целевая дата парсинга: 2025-05-18
[2025-05-18T20:33:03.596+0000] {docker.py:429} INFO - [2025-05-18 20:33:03] [INFO] [Parser] - Попытка подключения к БД (1/5): host=postgres_results_db_service port=5432 dbname=parsing_results user=parser_user
[2025-05-18T20:33:03.600+0000] {docker.py:429} INFO - [2025-05-18 20:33:03] [INFO] [Parser] - Успешное подключение к БД!
[2025-05-18 20:33:03] [INFO] [Parser] - Проверка/создание таблицы 'telegram_messages'...
[2025-05-18T20:33:03.603+0000] {docker.py:429} INFO - [2025-05-18 20:33:03] [INFO] [Parser] - Схема БД 'telegram_messages' проверена/обновлена.
[2025-05-18 20:33:03] [INFO] [Parser] - Инициализация Telegram клиента с сессией: /app/session/my_telegram_session
[2025-05-18T20:33:03.603+0000] {docker.py:429} INFO - [2025-05-18 20:33:03] [INFO] [Parser] - Файл сессии '/app/session/my_telegram_session.session' существует: True
[2025-05-18T20:33:03.604+0000] {docker.py:429} INFO - [2025-05-18 20:33:03] [INFO] [Parser] - Подключение к Telegram...
[2025-05-18 20:33:03] [INFO] [Parser] - Connecting to 149.154.167.51:443/TcpFull...
[2025-05-18 20:33:03] [DEBUG] [Parser] - Connection attempt 1...
[2025-05-18T20:33:03.852+0000] {docker.py:429} INFO - [2025-05-18 20:33:03] [DEBUG] [Parser] - Connection success!
[2025-05-18 20:33:03] [DEBUG] [Parser] - Starting send loop
[2025-05-18 20:33:03] [DEBUG] [Parser] - Starting receive loop
[2025-05-18 20:33:03] [INFO] [Parser] - Connection to 149.154.167.51:443/TcpFull complete!
[2025-05-18T20:33:03.856+0000] {docker.py:429} INFO - [2025-05-18 20:33:03] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18 20:33:03] [DEBUG] [Parser] - Assigned msg_id = 7505886494884557812 to InvokeWithLayerRequest (ffd72f4fdcd0)
[2025-05-18T20:33:03.856+0000] {docker.py:429} INFO - [2025-05-18 20:33:03] [DEBUG] [Parser] - Encrypting 1 message(s) in 80 bytes for sending
[2025-05-18T20:33:03.856+0000] {docker.py:429} INFO - [2025-05-18 20:33:03] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18 20:33:03] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18 20:33:03] [DEBUG] [Parser] - Receiving items from the network...
[2025-05-18T20:33:04.103+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Updated time offset (old offset 0, bad 7505886496163221092, good 7505886495436841985, new -1)
[2025-05-18 20:33:04] [DEBUG] [Parser] - Handling bad salt for message 7505886494884557812
[2025-05-18 20:33:04] [DEBUG] [Parser] - 1 message(s) will be resent
[2025-05-18 20:33:04] [DEBUG] [Parser] - Receiving items from the network...
[2025-05-18 20:33:04] [DEBUG] [Parser] - Assigned msg_id = 7505886491870124904 to InvokeWithLayerRequest (ffd72f4fdcd0)
[2025-05-18T20:33:04.103+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypting 1 message(s) in 80 bytes for sending
[2025-05-18T20:33:04.104+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18 20:33:04] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18 20:33:04] [DEBUG] [Parser] - Assigned msg_id = 7505886491871926392 to MsgsAck (ffd72e5030a0)
[2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypting 1 message(s) in 36 bytes for sending
[2025-05-18T20:33:04.104+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18 20:33:04] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18T20:33:04.352+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Handling container
[2025-05-18 20:33:04] [DEBUG] [Parser] - Handling new session created
[2025-05-18 20:33:04] [DEBUG] [Parser] - Handling acknowledge for [7505886491870124904]
[2025-05-18 20:33:04] [DEBUG] [Parser] - Receiving items from the network...
[2025-05-18T20:33:04.380+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Handling RPC result for message 7505886491870124904
[2025-05-18T20:33:04.381+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Receiving items from the network...
[2025-05-18T20:33:04.381+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Assigned msg_id = 7505886492984385576 to GetUsersRequest (ffd72e5032e0)
[2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypting 1 message(s) in 32 bytes for sending
[2025-05-18T20:33:04.381+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18 20:33:04] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18 20:33:04] [DEBUG] [Parser] - Assigned msg_id = 7505886492985101788 to MsgsAck (ffd72f784130)
[2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypting 1 message(s) in 60 bytes for sending
[2025-05-18T20:33:04.381+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18 20:33:04] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18T20:33:04.654+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Handling RPC result for message 7505886492984385576
[2025-05-18T20:33:04.654+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Receiving items from the network...
[2025-05-18T20:33:04.655+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Assigned msg_id = 7505886494081333248 to GetStateRequest (ffd72f4fdcd0)
[2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypting 1 message(s) in 20 bytes for sending
[2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18T20:33:04.655+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18T20:33:04.656+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Assigned msg_id = 7505886494083207216 to MsgsAck (ffd72e5b9df0)
[2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypting 1 message(s) in 36 bytes for sending
[2025-05-18T20:33:04.656+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18 20:33:04] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18T20:33:04.912+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Handling RPC result for message 7505886494081333248
[2025-05-18T20:33:04.912+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Receiving items from the network...
[2025-05-18T20:33:04.913+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Assigned msg_id = 7505886495110095112 to GetDifferenceRequest (ffd72f4fdcd0)
[2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypting 1 message(s) in 36 bytes for sending
[2025-05-18T20:33:04.913+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18T20:33:04.913+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18 20:33:04] [DEBUG] [Parser] - Assigned msg_id = 7505886495112488832 to MsgsAck (ffd72e503700)
[2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypting 1 message(s) in 36 bytes for sending
[2025-05-18T20:33:04.913+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18T20:33:04.914+0000] {docker.py:429} INFO - [2025-05-18 20:33:04] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18T20:33:05.240+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Handling RPC result for message 7505886495110095112
[2025-05-18 20:33:05] [DEBUG] [Parser] - Receiving items from the network...
[2025-05-18T20:33:05.241+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Assigned msg_id = 7505886496718788832 to GetUsersRequest (ffd72e5b9d30)
[2025-05-18 20:33:05] [DEBUG] [Parser] - Encrypting 1 message(s) in 32 bytes for sending
[2025-05-18T20:33:05.241+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18T20:33:05.242+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18 20:33:05] [DEBUG] [Parser] - Assigned msg_id = 7505886496722418516 to MsgsAck (ffd72e503c70)
[2025-05-18 20:33:05] [DEBUG] [Parser] - Encrypting 1 message(s) in 36 bytes for sending
[2025-05-18T20:33:05.242+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18 20:33:05] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18T20:33:05.602+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Handling RPC result for message 7505886496718788832
[2025-05-18T20:33:05.603+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Receiving items from the network...
[2025-05-18T20:33:05.603+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [INFO] [Parser] - Успешно подключен к Telegram как Ильяс (ID: 1257870016).
[2025-05-18 20:33:05] [INFO] [Parser] - Начало парсинга канала 'https://t.me/trader_chernyh' за дату 2025-05-18
[2025-05-18 20:33:05] [INFO] [Parser] - Получение информации о канале 'https://t.me/trader_chernyh'...
[2025-05-18T20:33:05.603+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Assigned msg_id = 7505886498169239728 to ResolveUsernameRequest (ffd72e503520)
[2025-05-18 20:33:05] [DEBUG] [Parser] - Encrypting 1 message(s) in 40 bytes for sending
[2025-05-18T20:33:05.603+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18T20:33:05.603+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18 20:33:05] [DEBUG] [Parser] - Assigned msg_id = 7505886498170355528 to MsgsAck (ffd72e5b9d90)
[2025-05-18T20:33:05.603+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Encrypting 1 message(s) in 36 bytes for sending
[2025-05-18T20:33:05.604+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18 20:33:05] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18T20:33:05.875+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Handling RPC result for message 7505886498169239728
[2025-05-18T20:33:05.875+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Receiving items from the network...
[2025-05-18T20:33:05.875+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [INFO] [Parser] - Канал найден: 'Евгений Черных и 401🍋' (ID: 1137503644)
[2025-05-18 20:33:05] [INFO] [Parser] - Итерация сообщений с offset_date=2025-05-19T00:00:00+00:00...
[2025-05-18T20:33:05.876+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Assigned msg_id = 7505886499258455008 to GetHistoryRequest (ffd72e503e20)
[2025-05-18 20:33:05] [DEBUG] [Parser] - Encrypting 1 message(s) in 72 bytes for sending
[2025-05-18T20:33:05.876+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18 20:33:05] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18 20:33:05] [DEBUG] [Parser] - Assigned msg_id = 7505886499259452552 to MsgsAck (ffd72e503dc0)
[2025-05-18 20:33:05] [DEBUG] [Parser] - Encrypting 1 message(s) in 36 bytes for sending
[2025-05-18T20:33:05.876+0000] {docker.py:429} INFO - [2025-05-18 20:33:05] [DEBUG] [Parser] - Encrypted messages put in a queue to be sent
[2025-05-18 20:33:05] [DEBUG] [Parser] - Waiting for messages to send...
[2025-05-18T20:33:06.880+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [DEBUG] [Parser] - Handling RPC result for message 7505886499258455008
[2025-05-18T20:33:06.897+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [DEBUG] [Parser] - Receiving items from the network...
[2025-05-18T20:33:06.899+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [INFO] [Parser] - Достигнуто начало дня 2025-05-18. Завершение итерации.
[2025-05-18 20:33:06] [INFO] [Parser] - Парсинг завершен. Получено сообщений: 2
[2025-05-18 20:33:06] [INFO] [Parser] - Список сообщений развернут.
[2025-05-18 20:33:06] [INFO] [Parser] - Успешно получено 2 сообщений из Telegram.
[2025-05-18 20:33:06] [INFO] [Parser] - Начало сохранения 2 сообщений в БД...
[2025-05-18T20:33:06.900+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [DEBUG] [Parser] - Не удалось декодировать bytes (длина: 29) как UTF-8, используется base64.
[2025-05-18 20:33:06] [DEBUG] [Parser] - Не удалось декодировать bytes (длина: 148) как UTF-8, используется base64.
[2025-05-18T20:33:06.900+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [DEBUG] [Parser] - Данные для вставки (ID: 9668): {'message_id': 9668, 'channel_id': 1137503644, 'message_date': datetime.datetime(2025, 5, 18, 9, 30, 40, tzinfo=datetime.timezone.utc), 'text': '**Ну что, коллеги, котлету зажарили? ****😁****\n**\nПомните, как на прошлой неделе говорил: **уровень 2780 по индексу надо выкупать?**\n\n✅ Уровень устоял. Кто успел — красавчик. \n\n**Можете смело брать с полки пирожок. С вареньем, не с капустой.\n**\nНО! Мы отскочили. И я уже вижу зону, где могут начать падать акции.\n\nВ новом ролике рассказал об этом 👇\n\n[**➡️ Ютуб**](https://youtu.be/5AwSbmokQz8)\n\n[**➡️ Рутуб**](https://rutube.ru/video/5ee7f6f9a8012d868d2e43ecf69c6ba2/)\n\n[**➡️ ВК.видео**](https://vk.com/trader_chernyh?z=video-98605354_456239285%2F2864508e17832f796a%2Fpl_wall_314989763)', 'sender_id': -1001137503644, 'views': 22681, 'forwards': 37, 'is_reply': False, 'reply_to_msg_id': None, 'has_media': True, 'raw_data_type': 'Json'}
[2025-05-18T20:33:06.901+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [DEBUG] [Parser] - Не удалось декодировать bytes (длина: 29) как UTF-8, используется base64.
[2025-05-18 20:33:06] [DEBUG] [Parser] - Не удалось декодировать bytes (длина: 136) как UTF-8, используется base64.
[2025-05-18 20:33:06] [DEBUG] [Parser] - Не удалось декодировать bytes (длина: 29) как UTF-8, используется base64.
[2025-05-18 20:33:06] [DEBUG] [Parser] - Не удалось декодировать bytes (длина: 137) как UTF-8, используется base64.
[2025-05-18 20:33:06] [DEBUG] [Parser] - Не удалось декодировать bytes (длина: 29) как UTF-8, используется base64.
[2025-05-18T20:33:06.901+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [DEBUG] [Parser] - Не удалось декодировать bytes (длина: 29) как UTF-8, используется base64.
[2025-05-18 20:33:06] [DEBUG] [Parser] - Не удалось декодировать bytes (длина: 137) как UTF-8, используется base64.
[2025-05-18T20:33:06.902+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [DEBUG] [Parser] - Не удалось декодировать bytes (длина: 29) как UTF-8, используется base64.
[2025-05-18T20:33:06.902+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [DEBUG] [Parser] - Данные для вставки (ID: 9669): {'message_id': 9669, 'channel_id': 1137503644, 'message_date': datetime.datetime(2025, 5, 18, 13, 25, 12, tzinfo=datetime.timezone.utc), 'text': 'На бирже у нас сейчас ключевой момент и вот почему:\n\n1️⃣ Технически идет завершение треугольника. И отсюда будет молниеносный выход! **__На принятие решения у Вас осталась одна неделя.__**\n\nГляньте ролик выше ⬆️ \n\n2️⃣ Забавно, что это совпадает с долгожданными переговорами в Стамбуле.  Тот случай, когда техника на графиках рулит политикой.\n\nВот ссылки на обзоры. Срочно смотрим и готовимся! ⬇️\n\n[**➡️ Ютуб**](https://youtu.be/5AwSbmokQz8)\n\n[**➡️ Рутуб**](https://rutube.ru/video/5ee7f6f9a8012d868d2e43ecf69c6ba2/)\n\n[**➡️ ВК.видео**](https://vk.com/trader_chernyh?z=video-98605354_456239285%2F2864508e17832f796a%2Fpl_wall_314989763)\n\n__Забавно, что точно такой же тезис после просмотра моего ролика, растащили себе инфоцыгане. Гопоте на пиво не хватает)__ __Который__ __раз воруют идеи__', 'sender_id': -1001137503644, 'views': 18774, 'forwards': 36, 'is_reply': False, 'reply_to_msg_id': None, 'has_media': True, 'raw_data_type': 'Json'}
[2025-05-18T20:33:06.902+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [INFO] [Parser] - Сохранение в БД завершено. Новых записей: 0, пропущено (ошибки/дубликаты): 2.
[2025-05-18 20:33:06] [INFO] [Parser] - Результаты сохранения в БД: 0 новых, 2 пропущено.
[2025-05-18 20:33:06] [ERROR] [Parser] - Ни одно из полученных сообщений не было сохранено в БД из-за ошибок.
[2025-05-18T20:33:06.903+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [ERROR] [Parser] - Критическая ошибка в main: Критическая ошибка: не удалось сохранить сообщения в БД.
Traceback (most recent call last):
  File "/app/telegram_parser.py", line 376, in main
    raise Exception("Критическая ошибка: не удалось сохранить сообщения в БД.")
Exception: Критическая ошибка: не удалось сохранить сообщения в БД.
[2025-05-18 20:33:06] [INFO] [Parser] - Запись данных XCom в файл: /airflow/xcom/return.json
[2025-05-18T20:33:06.903+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [INFO] [Parser] - Создана директория для XCom: /airflow/xcom
[2025-05-18T20:33:06.903+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [INFO] [Parser] - Данные XCom успешно записаны: {'channel_id': 1137503644, 'target_date_str': '2025-05-18', 'parsed_count': 2, 'saved_count': 0, 'status': 'failure', 'error_type': None, 'error_message': 'Критическая ошибка: не удалось сохранить сообщения в БД.'}
[2025-05-18T20:33:06.903+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [INFO] [Parser] - Отключение от Telegram...
[2025-05-18T20:33:06.904+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [INFO] [Parser] - Disconnecting from 149.154.167.51:443/TcpFull...
[2025-05-18 20:33:06] [DEBUG] [Parser] - Closing current connection...
[2025-05-18T20:33:06.904+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [DEBUG] [Parser] - Cancelling 0 pending message(s)...
[2025-05-18 20:33:06] [INFO] [Parser] - Disconnection from 149.154.167.51:443/TcpFull complete!
[2025-05-18T20:33:06.908+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [INFO] [Parser] - Соединение с Telegram закрыто.
[2025-05-18 20:33:06] [INFO] [Parser] - Закрытие соединения с БД...
[2025-05-18T20:33:06.908+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [INFO] [Parser] - Соединение с БД закрыто.
[2025-05-18 20:33:06] [INFO] [Parser] - --- Скрипт парсера завершен с кодом выхода: 1 ---
[2025-05-18T20:33:06.909+0000] {docker.py:429} INFO - [2025-05-18 20:33:06] [INFO] [Parser] - Перехвачен SystemExit с кодом 1. Завершение (из __main__).
[2025-05-18 20:33:06] [INFO] [Parser] - Скрипт парсера окончательно завершается с кодом 1 (из __main__ блока).
[2025-05-18T20:33:07.129+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/docker/operators/docker.py", line 502, in execute
    return self._run_image()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/docker/operators/docker.py", line 376, in _run_image
    return self._run_image_with_mounts(self.mounts, add_tmp_variable=False)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/docker/operators/docker.py", line 437, in _run_image_with_mounts
    raise DockerContainerFailedException(f"Docker container failed: {result!r}", logs=log_lines)
airflow.providers.docker.exceptions.DockerContainerFailedException: Docker container failed: {'StatusCode': 1}
[2025-05-18T20:33:07.131+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=telegram_summary_channel_1137503644, task_id=parse_channel_posts, execution_date=20250518T200000, start_date=20250518T203303, end_date=20250518T203307
[2025-05-18T20:33:07.141+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 61 for task parse_channel_posts (Docker container failed: {'StatusCode': 1}; 285)
[2025-05-18T20:33:07.174+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-05-18T20:33:07.190+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
